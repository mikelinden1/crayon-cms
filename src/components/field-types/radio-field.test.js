import RadioField from './radio-field';

const props = {
    name: 'test',
    value: 'val1',
    options: [
        { label: 'val1', value: 'val1'},
        { label: 'val2', value: 'val2'},
        { label: 'val3', value: 'val3'}
    ],
    onChange: () => {},
    disabled: false
};

it('RadioField matches snapshot', () => {
    const wrapper = shallow(<RadioField {...props} />);
    expect(wrapper).toMatchSnapshot();
});
