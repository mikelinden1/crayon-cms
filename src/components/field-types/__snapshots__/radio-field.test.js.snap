// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioField matches snapshot 1`] = `
Array [
  <div
    className="custom-control custom-radio"
    key="val1-id-field"
>
    <input
        checked={true}
        className="custom-control-input"
        disabled={false}
        id="val1-id"
        name="test"
        onChange={[Function]}
        type="radio"
        value="val1"
    />
    <label
        className="custom-control-label"
        htmlFor="val1-id"
    >
        val1
    </label>
</div>,
  <div
    className="custom-control custom-radio"
    key="val2-id-field"
>
    <input
        checked={false}
        className="custom-control-input"
        disabled={false}
        id="val2-id"
        name="test"
        onChange={[Function]}
        type="radio"
        value="val2"
    />
    <label
        className="custom-control-label"
        htmlFor="val2-id"
    >
        val2
    </label>
</div>,
  <div
    className="custom-control custom-radio"
    key="val3-id-field"
>
    <input
        checked={false}
        className="custom-control-input"
        disabled={false}
        id="val3-id"
        name="test"
        onChange={[Function]}
        type="radio"
        value="val3"
    />
    <label
        className="custom-control-label"
        htmlFor="val3-id"
    >
        val3
    </label>
</div>,
]
`;
